addon/index.js

```diff
@@ -1,25 +1,25 @@
-import { invokeHelper } from '@ember/helper';
```

Comment:
Revert these whitespace changes I guess.

Reply:
I have some editor things to update

Reply:
Apparently I had a rogue prettier plugin that wasn't specified anywhere, but still being used in my editor. O.o